src/runtime_src/core/common/xrt_graph.h

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (C) 2020, Xilinx Inc - All rights reserved
+ * Xilinx Runtime (XRT) Experimental APIs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may
+ * not use this file except in compliance with the License. A copy of the
+ * License is located at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+// This file defines shim level XRT Graph APIs.
+
+#ifndef _XRT_COMMON_GRAPH_H_
+#define _XRT_COMMON_GRAPH_H_
+
+typedef void * xclGraphHandle;
+
+xclGraphHandle
+xclGraphOpen(xclDeviceHandle handle, const uuid_t xclbinUUID, const char *graphName);
```

Comment:
Looks like xrt_graph.h is included in ishim.h which in turn is included by windows.  uuid_t will not work, use xuid_t.

Reply:
done